pdc/routers.py

```diff
@@ -74,7 +74,8 @@
                 compose_views.ComposeRPMMappingView,
                 base_name='composerpmmapping')
 router.register(r'compose-rpms', compose_views.ComposeRPMView)
-router.register(r'compose-images', compose_views.ComposeImageView)
+router.register(r'compose-images', compose_views.ComposeImageView,
+                base_name='composeimages')
```

Comment:
need to update base_name to `composeimage`, same as BuildImage. Otherwise related unit tests can't be pass.